lib/_tls_common.js

```diff
@@ -137,20 +137,34 @@ exports.createSecureContext = function createSecureContext(options, context) {
   }
 
   if (options.pfx) {
-    var pfx = options.pfx;
-    var passphrase = options.passphrase;
-
     if (!crypto)
       crypto = require('crypto');
-
-    pfx = crypto._toBuf(pfx);
-    if (passphrase)
-      passphrase = crypto._toBuf(passphrase);
-
-    if (passphrase) {
-      c.context.loadPKCS12(pfx, passphrase);
+    const Buffer = require('buffer').Buffer;
+
+    if (Array.isArray(options.pfx)) {
+      for (i = 0; i < options.pfx.length; i++) {
+        const pfx = options.pfx[i];
+        let buf;
+        if (pfx.buffer instanceof Buffer) {
```

Comment:
Also, should this be limited to just `Buffer` instances? We likely should allow any `Uint8Array`